packages/client/src/context/AuthContext.js

```diff
@@ -13,13 +13,25 @@ export const AuthContextProvider = ({ children }) => {
     return JSON.parse(localStorage.getItem("Token Object")) || false;
   });
 
+  const [resErrors, setResErrors] = useState(false);
+
   const login = useCallback(async ({ email, password }) => {
-    const result = await axios.post(`${backend}api/auth/login`, {
-      email,
-      password,
-    });
-    _dispatch(result.data);
-    localStorage.setItem("Token Object", JSON.stringify(result.data));
+    await axios
+      .post(`${backend}api/auth/login`, {
+        email,
+        password,
+      })
+      .then((response) => {
+        _dispatch(response.data);
+        localStorage.setItem("Token Object", JSON.stringify(response.data));
+      })
```

Comment:
We probably want to set `resErrors` to a negative state  on a second attempt. In this case `false` since that's what you have on line 16.

Reply:
I see, so, user may have a failed attempt, resErrors becomes set to the error response message; then user logs in successfully and we ensure that resErrors is now reset to false.  I believe I've done this correctly in new changes :)